src/parser/rogue/outlaw/modules/spells/BetweenTheEyes.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import Analyzer from 'parser/core/Analyzer';
+
+import SpellUsable from '../../../shared/SpellUsable';
+
+class BetweenTheEyes extends Analyzer {
+  static dependencies = {
+    spellUsable: SpellUsable,
+  };
+
+  constructor(...args) {
+    super(...args);
+    this.active = !this.selectedCombatant.hasTalent(SPELLS.SLICE_AND_DICE_TALENT.id) || this.selectedCombatant.hasTrait(SPELLS.ACE_UP_YOUR_SLEEVE.id) || this.selectedCombatant.hasTrait(SPELLS.DEADSHOT.id);
+  }
+
+  delayedCasts = 0;
+  
+  get suggestionThresholds() {
+    return {
+      actual: this.delayedCasts,
+      isGreaterThan: {
+        minor: 0,
+        average: 4,
+        major: 9,
+      },
+      style: 'number',
+    };
+  }
+
+  get hasRelevantTrait(){
+    return this.selectedCombatant.hasTrait(SPELLS.ACE_UP_YOUR_SLEEVE.id) || this.selectedCombatant.hasTrait(SPELLS.DEADSHOT.id);
+  }
+
+  on_byPlayer_cast(event){
+    if(event.ability.guid !== SPELLS.DISPATCH.id || this.spellUsable.isOnCooldown(SPELLS.BETWEEN_THE_EYES.id)){
+      return;
+    }
+
+    if(this.hasRelevantTrait || this.selectedCombatant.hasBuff(SPELLS.RUTHLESS_PRECISION.id)){
```

Comment:
My intention here was just noting when you used the wrong damaging finisher. I figured if you used BTE rather than RTB when you had no buffs, that would be reflected looking at RTB uptime. And if you delayed RTB used to reroll so that you could cast BTE, that would show up in the suggestions around RTB efficiency.

Reply:
At the start of the fight, the first finisher you should use is RTB, and this module will always flag it as an error, or am I wrong?

Reply:
And in other cases your approach just does not work. If RTB just ran out, its perfectly valid to use RTB over BTE. And this module spots it as an error. Where as if someone decided to incorrectly use BTE they now get punished for low RTB uptime. Basically, your analysis seems to have no correct cast in this situation.

Reply:
This handler only keeps casts that are Dispatch, so any RTB casts won't land in here anyway. This will only flag an error if you use Dispatch when you could have used BTE

Reply:
you are right that wors